microsoft-edge/webview2/concepts/security.md

```diff
@@ -0,0 +1,35 @@
+---
+description: Learn about how to build a secure WebView2 application
+title: WebView2 Security Best Practicies
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 05/21/2020
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html, security
+---
+
+# Security Best Practices
+
+The WebView2 control hosts web content. While there are many advantages of hosting web content inside your native application, when used incorrectly, web content can create vulnerabilities. Therefore, we recommend designing your WebView2 appication to closely monitor the interactions between the web content and the host application in line with the following practices.
+
+## Best Practices
+
+1. Treat all web content as insecure
+    - Validate web messages and host object parameters before consuming them, because web messages and parameters can be malformed (unintentionally or maliciously) and cause the app to behave unexpectedly.
```

Comment:
```suggestion
* To prevent using malformed web messages and parameters, validate web messages and host object parameters to prevent your application from behaving unexpectedly. 
```

Reply:
Can we include a link to an example of how to do this? Or include a code snippet?

Reply:
Not really a standard way to do this, based on the usage of the developer